backend/routes/api.php

```diff
@@ -35,5 +35,5 @@
     Route::apiResource('users.accounts', AccountController::class)->shallow()->only(['index']);
     Route::apiResource('accounts', AccountController::class)->only(['index', 'show']);
     Route::apiResource('accounts.transactions', TransactionController::class)->shallow()->only(['index']);
-    Route::apiResource('transactions', TransactionController::class)->only(['index', 'show']);
+    Route::apiResource('transactions', TransactionController::class)->only(['index', 'show', 'store']);
```

Comment:
use nested routing here

route POST accounts/{account_id}/transaction 
is much more sensible